packages/table/src/TableHeaderCell.tsx

```diff
@@ -49,21 +53,35 @@ const TableHeaderCell = <T extends IdType>({
   };
 
   const getHeaderColumnProps = (column: ExtendedTableHeader<T>) => {
+    const inheritProps = column.getHeaderProps();
     const props = {
       className: classNames(column.className || '', {
-        'fixed-width-text': scrollable && !column.className,
         sticky: column.stickyRight || column.stickyLeft,
         'sticky-right': column.stickyRight,
         'sticky-left': column.stickyLeft,
       }),
-      title: column.label || typeof column.Header === 'string' ? column.Header?.toString() : undefined,
+      title: undefined,
+      style: useColumnWidths
+        ? {
+            width: column.width,
+            minWidth: column.width,
+            maxWidth: column.width,
```

Comment:
**QUESITON/SUGGESTION:** Shouldn't we be setting these to the matching style props or otherwise omit setting the min/max altogether?
```suggestion
            minWidth: column.minWidth,
            maxWidth: column.maxWidth,
```